src/commands/chat/test-command.ts

```diff
@@ -2,14 +2,14 @@ import { RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-typ
 import { CommandInteraction, PermissionString } from 'discord.js';
 import { RateLimiter } from 'discord.js-rate-limiter';
 
-import { EventData } from '../models/internal-models.js';
-import { Lang } from '../services/index.js';
-import { InteractionUtils } from '../utils/index.js';
-import { Command, CommandDeferType } from './index.js';
+import { EventData } from '../../models/internal-models.js';
+import { Lang } from '../../services/lang.js';
+import { InteractionUtils } from '../../utils/interaction-utils.js';
+import { Command, CommandDeferType } from '../index.js';
 
 export class TestCommand implements Command {
     public metadata: RESTPostAPIChatInputApplicationCommandsJSONBody = {
-        name: Lang.getCom('commands.test'),
+        name: Lang.getCom('chatCommands.test'),
         description: Lang.getRef('commandDescs.test', Lang.Default),
         dm_permission: true,
```

Comment:
Can you add this to the message/user commands as well so it's explicit that it can be used in DMs?

Reply:
Current functionality of the user command won't work but I can for the message